### PR TITLE
#150704027 Build out dashboard for failed cronjobs

### PR DESCRIPTION
**Pull request by ogomb**
#### What does this PR do?
A dashboard for the failed cron jobs
Front end test for the failed cron jobs.

#### How should this be manually tested?
in order to run the test run the following code `python manage.py test hc/front/tests`

#### Any background context you want to provide?
The application healthchecks is a watchdog for your cron jobs. It's a web server that listens for pings from your cron jobs, plus a web interface. This PR builds on that by creating a dashboard for the jobs that are down/failed.

#### Screenshots 

dashboard_with failed checks

empty_failed dashborard

#### What are the relevant pivotal tracker stories?
#150704027

### CODE REVIEW

File: hc/front/views.py [hunk 1]
```diff
@@ -60,6 +60,28 @@ def my_checks(request):
     return render(request, "front/my_checks.html", ctx)
 
 
+@login_required
+def failed_checks(request):
+    q = Check.objects.filter(user=request.team.user).order_by("created")
```
Comment (collinmutembei) [inline hc/front/views.py]: We talked about this, use descriptive variable names

File: hc/front/views.py [hunk 2]
```diff
@@ -60,6 +60,28 @@ def my_checks(request):
...
+    checks = list(q)
```
Comment (collinmutembei) [inline hc/front/views.py]: Why convert `q` to a list?

**Review by collinmutembei — CHANGES_REQUESTED** [review]

**Comment by collinmutembei** [comment]
Also, include screenshots for the changes in this feature

File: hc/front/views.py (hunk 1 above)
Comment (ogomb) [reply hc/front/views.py]: Our bad we are changing the code...

File: hc/front/views.py (hunk 2 above)
Comment (ogomb) [reply hc/front/views.py]: At first we did not realise that the Queryset object returned from the query was actually a list. So we were converting it to a list in order to loop over it.

We now understand it works like a list.

Comment (collinmutembei) [reply hc/front/views.py]: Good, update it to iterate over the [queryset](https://docs.djangoproject.com/en/1.11/ref/models/querysets)

**Review by collinmutembei — APPROVED** [review]

**Comment by collinmutembei** [comment]
Deploy to the [dev](https://hc-f-society-dev.herokuapp.com/) environment, so we can have a look 👀

**Pull request merged**